scripts/update-deps.sh

```diff
@@ -25,5 +25,5 @@ get boostorg/type_traits boost-1.56.0 boost/type_traits
 get boostorg/typeof boost-1.56.0 boost/typeof
 get boostorg/utility boost-1.56.0 boost/utility
 get joyent/http-parser v2.1-47-g1b31580 http-parser
-get bassosimone/libevent release-2.0.21-stable-52-g86b586a libevent
+get measurement-kit/measurement-kit-libevent release-2.0.22-stable-15-ged01cec libevent
```

Comment:
@DavideAllavena could you please update the name of the repository here? It was renamed and now it's called just libevent. Thanks!